docs/tsc/meetings/2019-02-04.md

```diff
@@ -0,0 +1,36 @@
+
+Feb. 4, 2019
+
```

Comment:
I think that the document should mention the host/organizer:
"Host: John Mertic"